core/src/main/scala/org/apache/spark/scheduler/SchedulableBuilder.scala

```diff
@@ -83,15 +83,19 @@ private[spark] class FairSchedulableBuilder(val rootPool: Pool, conf: SparkConf)
           Some((is, DEFAULT_SCHEDULER_FILE))
         } else {
           logWarning("Fair Scheduler configuration file not found so jobs will be scheduled " +
-            "in FIFO order")
+            s"in FIFO order. To use fair scheduling, configure pools in $DEFAULT_SCHEDULER_FILE " +
+            "or set spark.scheduler.allocation.file to a file that contains the configuration.")
```

Comment:
one last tiny nit: can you make a varaibel name spark.scheduler.allocaiton.file and use it here + above?